Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN go mod download
 
 COPY . .
 
-RUN go build -o /usr/local/bin/vault-operator ./cmd/manager/
+RUN go build -o /usr/local/bin/vault-operator ./cmd/
```

Comment:
Isn't cmd/manager the standard for kubebuilder/operators?

Reply:
no, the `cmd/` is

Reply:
Understood